stderr of test 'remote11` in directory 'monetdb5/modules/mal` itself:


# 11:48:19 >  
# 11:48:19 >   mserver5  --debug=10 --set gdk_nr_threads=0  --set "gdk_dbfarm=/net/volund.ins.cwi.nl/export/scratch0/fabian/vtmp/mtest-current-volund.ins.cwi.nl/five/dbfarm" --set mapi_open=true --set mapi_port=30094 --set monet_prompt= --trace --forcemito --set mal_listing=2  --dbname=mTests_modules_mal  remote11.mal
# 11:48:19 >  

# builtin opt 	gdk_dbname = demo
# builtin opt 	gdk_dbfarm = /ufs/fabian/scratch/monetdb/current/program-x86_64/var/lib/monetdb5/dbfarm
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_alloc_map = no
# builtin opt 	gdk_vmtrim = yes
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	gdk_dbfarm = /net/volund.ins.cwi.nl/export/scratch0/fabian/vtmp/mtest-current-volund.ins.cwi.nl/five/dbfarm
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 30094
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbname = mTests_modules_mal

# 11:48:21 >  
# 11:48:21 >  Done.
# 11:48:21 >  

